pyrefly/lib/test/lsp/completion.rs

```diff
@@ -1295,8 +1296,8 @@ foo('
 4 | foo('
          ^
 Completion Results:
-- (Value) 'bar': Literal['bar']
-- (Value) 'foo': Literal['foo']
+- (Value) 'bar': Literal['bar'] inserting `bar`
```

Comment:
it seems like this case is identifier as in a string literal?

is it possible to add a test for `foo(` (no quotes at all) where we see `inserting `"bar"`` so it's clear that the lack of quotes mean it's not inserting quotes?